packages/authentication/src/definitions.ts

```diff
@@ -667,6 +673,38 @@ export interface GetIdTokenResult {
   token: string;
 }
 
+export interface GetIdTokenInfo {
```

Comment:
```suggestion
export interface GetIdTokenResultResult {
```

You should also rename the interface.